index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">  
+    <title>Personal Portfolio</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header class="menu">
+            <div class="menu-logo">
+                <h1>Welcome</h1>
+            </div>
```

Comment:
- [ ] Nice you have a logo named welcome and it is on the left side but it should be a link not h1
        * The logo (if the design has one) is on the left, and has a link (a tag)